litmus-portal/frontend/src/components/VerifyCommit/index.tsx

```diff
@@ -96,8 +96,9 @@ function VerifyCommit() {
                     handleClick={() => {
                       // console.log('Edit');
                     }}
-                    value="Edit"
-                  />
+                  >
+                    <Typography>Edit</Typography>
```

Comment:
Typography shouldn't be required as the component provides that already

Reply:
* [x] Fixed